### PR TITLE
Monitor for postgres table sizes.

### PR DESCRIPTION
**Pull request by tasdomas**

### CODE REVIEW

File: table_size.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 Canonical Ltd.
```
Comment (mattyw) [inline table_size.go]: 2017

File: table_size_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// Copyright 2016 Canonical Ltd.
```
Comment (mattyw) [inline table_size_test.go]: 2017

File: table_size_test.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package monitoring_test
+
+import (
+	"database/sql"
+
+	"github.com/CanonicalLtd/omniutils/testing/pgtest"
+	//"github.com/juju/testing"
```
Comment (mattyw) [inline table_size_test.go]: dd

File: table_size_test.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	jc "github.com/juju/testing/checkers"
+	"github.com/prometheus/client_golang/prometheus"
+	//prometheusinternal "github.com/prometheus/client_model/go"
```
Comment (mattyw) [inline table_size_test.go]: dd

**Review by mattyw — APPROVED** [review]

File: table_size.go (hunk 1 above)
Comment (alesstimec) [inline table_size.go]: s/2016/2017/

File: table_size_test.go (hunk 2 above)
Comment (alesstimec) [inline table_size_test.go]: s/2016/2017/

**Review by alesstimec — APPROVED** [review]
LGTM modulo two nitpicks

**Pull request merged**